src/cdn/combineCKCdnBundlesPacks.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import type { CKCdnResourcesPack, InferCKCdnResourcesPackExportsType } from './loadCKCdnResourcesPack';
+
+/**
+ * Combines multiple CKEditor CDN bundles packs into a single pack.
+ *
+ * @param packs A map of CKEditor CDN bundles packs.
+ * @returns A combined pack of resources for multiple CKEditor CDN bundles.
+ * @example
+ *
+ * ```ts
+ * const { Base, Premium } = await loadCKCdnResourcesPack(
+ * 	combineCKCdnBundlesPacks( {
+ * 		Base: createCKCdnBaseBundlePack( {
+ * 			version: '43.0.0',
+ * 			languages: [ 'en', 'de' ]
+ * 		} ),
+ * 		Premium: createCKCdnPremiumBundlePack( {
+ * 			version: '43.0.0',
+ * 			languages: [ 'en', 'de' ]
+ * 		} )
+ * 	} )
+ * );
+ * ```
+ */
+export function combineCKCdnBundlesPacks<P extends CKCdnBundlesPacks>( packs: P ): CKCdnCombinedBundlePack<P> {
+	const allPacks = (
+		Object
+			.values( packs )
+			.filter( Boolean )
+	) as Array<CKCdnResourcesPack<any>>;
```

Comment:
```suggestion
	const allPacks = Object
		.values( packs )
		.filter( pack => pack !== undefined );
```

Because this repo uses TypeScript 5.5, I think we can get the proper type without casting.

Example using TypeScript 5.5: [link](https://www.typescriptlang.org/play/?#code/MYewdgzgLgBAJgQyggXDASgU1AJzgHmhwEswBzAGhjAFcBbAI0xxgB8Yaw5MAzUzOAD4YAXhgBtALoBuAFCzQkWHwA2UZgHViUABYAhECBWYEYUTADyDAFbYoAOgBuCFTUwQAFPCQIYASntVdRwvAyMTMz85AHpomHj4gD0AfnlFaBggzW0dAFUuXn44AGEdbABrcytbYAdnV3cvRGR-QOI1Zi9MMCgcAE9RYW7egYBCETFObj4wAX8YuISU2SA).
Example using TypeScript 5.4: [link](https://www.typescriptlang.org/play/?ts=5.4.5#code/MYewdgzgLgBAJgQyggXDASgU1AJzgHmhwEswBzAGhjAFcBbAI0xxgB8Yaw5MAzUzOAD4YAXhgBtALoBuAFCzQkWHwA2UZgHViUABYAhECBWYEYUTADyDAFbYoAOgBuCFTUwQAFPCQIYASntVdRwvAyMTMz85AHpomHj4gD0AfnlFaBggzW0dAFUuXn44AGEdbABrcytbYAdnV3cvRGR-QOI1Zi9MMCgcAE9RYW7egYBCETFObj4wAX8YuISU2SA).

Reply:
Yep, thx!